CreateService/handler.ts

```diff
@@ -97,19 +100,34 @@ const createSubscriptionTask = (
         },
         subscription_id: subscriptionId
       }),
-    errs => ResponseErrorInternal(JSON.stringify(errs))
+    errs => {
+      context.log(
+        `${logPrefix}| createSubscriptionTask on call errs=${JSON.stringify(
```

Comment:
1. get rid of the space after `|`
2. try not to use implementation names / details which may be subject to refactors 
3. use ERROR= (or ERROR_DETALS= when using context.verbose)

```suggestion
        `${logPrefix}|CreateSubscription|ERROR=${JSON.stringify(
```

Reply:
Fixed in [d7f70c2](https://github.com/pagopa/io-functions-services/pull/68/commits/d7f70c2bb120fe7b97cf5a1bec35ce16ad528640)